source/class/qx/test/bom/Font.js

```diff
@@ -181,7 +189,7 @@ qx.Class.define("qx.test.bom.Font",
       var keys = Object.keys(styles);
 
       this.assertMap(styles, "Method 'getStyles' should return a map!");
-      this.assertEquals(8, qx.lang.Object.getLength(styles), "Map should contain 8 key!");
+      this.assertEquals(9, qx.lang.Object.getLength(styles), "Map should contain 8 key!");
```

Comment:
Error message should say "Map should contain 9 keys!" (changes both the number and fixes the plural of "key")

Reply:
Oops, missed that. Thanks!